src/zkevm_specs/opcode/slt_sgt.py

```diff
@@ -0,0 +1,55 @@
+from typing import Sequence
```

Comment:
only keep one file `src/zkevm_specs/evm/execution/slt_sgt.py` will be ok, this file can be removed